src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -72,9 +72,9 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $cliOutput)
```

Comment:
This is not named according to the parent definition. Also `inheritdoc` will not work this way.